src/components/definition-list/dd.d.ts

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+export interface DdProps {
+    /** Label margin bottom styling. This value will be multiplied by the theme spacing constant (8) */
+    mb?: 0 | 1 | 2 | 3 | 4 | 5 | 6;
+
+    /** Prop for what will render in the `<Dd></Dd>` tags */
+    children: string | React.ReactNode;
```

Comment:
This can just be `children: React.ReactNode;`

Reply:
✅